docker-compose/README.md

```diff
@@ -56,6 +161,67 @@ services:
 
 It is not required to link `./volumes/mysql` location. In case you are using some other location for persistent data, feel free to use it instead.
 
+## Importing data from pulse file
+
+Catalog QT Demonstrator allows to import MDSPlus based data automatically into SQL database. In order to do this you have to bind mount a volume. In a plain text it means that you have to tell Docker that you want to make your local filesystem to be available inside Docker container. Easiest way to do it is to create directory (or symbolic link) to a MDSPlus compatible local database.
+
+First of all, make sure you have `MDSPlus` like directory structure with pulse files. The easiest way to execute Docker container with sample data is to get sample data from `box.psnc.pl` - these are completely artificially created data prepared by testing framework.
+
+```
+> curl -s -o f4f_data.tar.gz \
```

Comment:
I get `Bad access token`

Reply:
and i got this:
![image](https://user-images.githubusercontent.com/34068433/125439722-ec16e81c-e230-4b66-b895-675961d17907.png)

@mkowsiak We need to correct this file on box

Reply:
The issue here is that box.psnc.pl seems to invalidate some locations (download links) even though they are marked as permanent :( To be honest I don't have a good idea how to fix that. We can either upload this file into git repository (I'd prefer to avoid it), we can try going with box.pionier.net.pl (I will try this approach), or we can use something else (e.g. DropBox or Google Drive).

Reply:
Can you give it a try now? With new command:

```
> curl -L -o f4f.tar "https://box.pionier.net.pl/f/f03c04fbac70477fb748/?dl=1"
```

Reply:
@mkowsiak  Now works :)